.env.example

```diff
@@ -30,4 +30,11 @@ BUILD_LOCALES=
 
 # If resource constraints are being hit during builds, change LIMIT_CPUS to a
 # fixed number of CPUs (e.g. 2) to limit the demand during build time
-LIMIT_CPUS=
\ No newline at end of file
+LIMIT_CPUS=
+
+# Discord environment (ID and token required for Discord webhook call)
+# DISCORD_ID=
+# DISCORD_TOKEN=
+
+# Github token used for fetching good first issues
+# ISSUES_GITHUB_TOKEN_READ_ONLY
```

Comment:
```suggestion
```

Reply:
We just need to add these here: https://github.com/ethereum/ethereum-org-website/settings/secrets/actions